app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import datetime
+import time
+
+
+def files_creator():
+    while True:
+        accurate_time = datetime.datetime.now()
+        accurate_time = (f"app-{str(accurate_time.hour)}"
+                         f"_{str(accurate_time.minute)}"
+                         f"_{str(accurate_time.second)}.log")
+
+        with open(accurate_time, "w") as f:
+            f.write(str(datetime.datetime.now()))
```

Comment:
In this case, you can use `accurate_time ` here instead of calling the `now` one more time